src/nrnmpi/nrnmpi_dynam.cpp

```diff
@@ -157,11 +157,22 @@ sprintf(pmes+strlen(pmes), "Is openmpi installed? If not in default location, ne
 	 */
 	sprintf(pmes, "Try loading libmpi\n");
 	handle = load_mpi("libmpi.so", pmes+strlen(pmes));
+
+    // like osx, check if user has provided library via MPI_LIB_NRN_PATH
```

Comment:
Nitpick.. I think something with the formatting is off?

Reply:
yeah - old files has tabs and it's mess to match the old formatting (and tabs are bad anyway). This will be done with clang-formatting after 8.0 release